src/main/java/seedu/partyplanet/logic/autocomplete/EEditAutocompleteUtil.java

```diff
@@ -0,0 +1,95 @@
+package seedu.partyplanet.logic.autocomplete;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_REMARK;
+
+import java.util.List;
+import java.util.Map;
+
+import javafx.collections.ObservableList;
+import seedu.partyplanet.commons.core.Messages;
+import seedu.partyplanet.commons.core.index.Index;
+import seedu.partyplanet.logic.commands.exceptions.CommandException;
+import seedu.partyplanet.logic.parser.ArgumentMultimap;
+import seedu.partyplanet.logic.parser.ArgumentTokenizer;
+import seedu.partyplanet.logic.parser.ParserUtil;
+import seedu.partyplanet.logic.parser.Prefix;
+import seedu.partyplanet.logic.parser.exceptions.ParseException;
+import seedu.partyplanet.model.Model;
+import seedu.partyplanet.model.event.Event;
+
+public class EEditAutocompleteUtil {
+
+    private static final String INDEX_NOT_SPECIFIED_MESSAGE = "Index not specified!";
+
+    /**
+     * Parses an edit command to autocomplete remark.
+     * @param arguments User's input command.
+     * @param model Model instance containing address book.
+     * @return String of new autocompleted command.
+     * @throws ParseException If the input command does not follow requirements.
+     * @throws CommandException If the input command is out of bounds.
+     */
+    public String parseCommand(String arguments, Model model) throws ParseException, CommandException {
+        requireNonNull(arguments);
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(arguments, PREFIX_DATE, PREFIX_REMARK);
```

Comment:
```suggestion
                ArgumentTokenizer.tokenize(arguments, PREFIX_NAME, PREFIX_DATE, PREFIX_REMARK);
```
Probably should include auto complete for name for events since we allow it for persons